push.bash

```diff
@@ -51,4 +51,17 @@ if [ "${DRAFT}" == "true" ]; then
   BUF_ARGS=("--draft" "${GITHUB_REF_NAME}")
 fi
 
+if [ -n "${CREATE_VISIBILITY}" ]; then
+  set +e
+  BUF_HELP_OUTPUT="$("${BUF_COMMAND}" push example --create --help 2>&1)" # need to verify this
```

Comment:
can we expand this comment a bit

Reply:
Sorry that was reminder for me in development. It is no longer needed.